action-allowedlist/approved.json

```diff
@@ -21,16 +21,28 @@
     },
     {
       "actionLink": "actions/checkout",
-      "actionVersion": "5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f"
+      "actionVersion": "5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f",
+      "tag": "v2.3.4"
     },
     {
       "actionLink": "actions/checkout",
       "actionVersion": "ec3a7ce113134d7a93b817d10a8272cb61118579",
       "tag": "v2.4.0"
     },
+    {
+      "actionLink": "actions/checkout",
+      "actionVersion": "2541b1294d2704b0964813337f33b291d3f8596b",
+      "tag": "v3.0.2"
+    },
     {
       "actionLink": "actions/setup-python",
-      "actionVersion": "3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df"
+      "actionVersion": "3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df",
+      "tag": "v2.2.1"
+    },
+    {
+      "actionLink": "actions/setup-python",
+      "actionVersion": "98f2ad02fd48d057ee3b4d4f66525b231c3e52b6",
+      "tag": "v3.1.2"
```

Comment:
Newer version has Poetry built-in, so that one doesn't need to run `Gr1N/setup-poetry` or manually manage caching.